src/common/components/AddButton.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import {Image, StyleSheet, View} from 'react-native';
+import {scale} from 'react-native-size-matters';
+
 const add_image = require('../../assets/images/tabBarIcons/addIcon.png');
```

Comment:
Always use 'All capital letters in snake case for constants'
i.e.
`const ADD_IMAGE = require('...')`